server/server.js

```diff
@@ -17,6 +17,7 @@ registerMiddleware(app);
 app.use('/api/v1', api);
 
 app.listen(port, () => {
+  // eslint-disable-next-line no-console
   console.log(`Server listening on port: ${port} in ${process.env.NODE_ENV} mode`);
```

Comment:
What are your thoughts about having a default `process.env.NODE_ENV`?

Reply:
I am not sure what the implications of that are

Reply:
could you enlighten me?

Reply:
If you don't set a value explicitly it will be `undefined` thus the above console log will log `${process.env.NODE_ENV}` as undefined for anyone who runs `npm install` followed by `npm run start`.

Reply:
I see that makes sense. I noticed it defaulted to `production` on heroku. It caused issues for me because it didn't install `devDependencies` as a result.

Reply:
Is it alright to add `set NODE_ENV=development`  as part of `npm run start`  ?

Reply:
No, setting it inside the application will not be ideal. The NODE_ENV will have already been read by the time npm run start is trying to set it. I think the best way is to instruct the user to at least set the environment variable NODE_ENV in the README.

Reply:
Okay I am modifying the README accordingly then